src/apps/answers/tests/test_answers.py

```diff
@@ -3564,9 +3591,8 @@ async def test_validate_multiinformant_assessment_fail_source_subject_not_found(
 
         response = await client.get(url)
 
-        assert response.status_code == http.HTTPStatus.OK
-        assert response.json()["result"]["valid"] is False
-        assert response.json()["result"]["code"] == "invalid_source_subject"
+        assert response.status_code == http.HTTPStatus.UNPROCESSABLE_ENTITY
+        assert response.json()["result"][0]["message"] == "field required"
 
```

Comment:
this test should continue returning OK, you should pass all parameters, but a wrong source subject Id

Reply:
Fixed in last commit